reposerver/repository/repository.go

```diff
@@ -696,89 +697,11 @@ func (s *Service) runManifestGenAsync(ctx context.Context, repoRoot, commitSHA,
 	if err == nil {
 		// Much of the multi-source handling logic is duplicated in resolveReferencedSources. If making changes here,
 		// check whether they should be replicated in resolveReferencedSources.
-		if q.HasMultipleSources {
```

Comment:
Extracted to a function so it can be reused in the comparison

Reply:
Realized resolveReferencedSources already does the same thing so I reverted that change